pom.xml

```diff
@@ -371,7 +371,7 @@
 			<dependency>
 				<groupId>com.thoughtworks.xstream</groupId>
 				<artifactId>xstream</artifactId>
-				<version>1.4.11.1</version>
+				<version>1.4.12</version>
```

Comment:
try out the changes with reference application

Reply:
thanks @ HerbertYiga  I tested it in the same way as all the other PRs that I perform, by executing the jetty: run command, with the legacyUI module install.

Reply:
Testing with the reference application would require you to run these modules: https://ci.openmrs.org/build/result/artifactUrlRedirect.action?planKey=REFAPP-OMODDISTRO&buildNumber=9987&artifactName=referenceapplication-addons-snapshot